src/SignalR/common/SignalR.Common/src/Microsoft.AspNetCore.SignalR.Common.csproj

```diff
@@ -25,9 +25,10 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
-    <Reference Include="Microsoft.Bcl.Json.Sources" />
-    <Reference Include="System.Buffers" />
-    <Reference Include="System.Runtime.CompilerServices.Unsafe" />
+    <Reference Include="System.Text.Json" />
+
+    <!-- System.Text.Json brings System.Buffers in transitively. No need for both here. -->
+    <SuppressBaselineReference Include="System.Buffers" />
```

Comment:
Can you move this into the block below? This will prevent suppressions from bleeding over into the next major or minor version.

Reply:
I'll try it.